docs/examples/event-exporter.md

```diff
@@ -0,0 +1,271 @@
+| image    | Docker image of the event exporter.                    | `apache/skywalking-kubernetes-event-exporter:latest` |.                                                                                                           | `"127.0.0.1:11800"`                       |
+| replicas | Number of event exporter pods.                         | `1`                                                  |
+| config   | Configuration of filters and exporters in YAML format. | `""`                                                 |
+
+Please note: if you ignore the `config` field, no filters or exporter will be created.
+
+This is because the EventExporter controller creates a configMap for all `config` values and
+attach the configMap to the event exporter container as configuration file.
+Ignoring the `config` field means an **empty** configuration file (with value `"""`) is provided to the event exporter.
+
+## Status
+
+| name              | description                                                                |
+|-------------------|----------------------------------------------------------------------------|
+| availableReplicas | Total number of available event exporter pods.                             |
+| conditions        | Latest available observations of the underlying deployment's current state |
+| configMapName     | Name of the underlying configMap.                                          |
+
+## Configuration
+
+The event exporter supports reporting specific events by different exporters.
+We can add filter configs to choose which events we are interested in,
+and include exporter names in each filter config to tell event exporter how to export filtered events.
+
+An example configuration is listed below:
+
+```yaml
+filters:
+  - reason: ""
+    message: ""
+    minCount: 1
+    type: ""
+    action: ""
+    kind: "Pod|Service"
+    namespace: "^default$"
+    name: ""
+    service: "[^\\s]{1,}"
+    exporters:
+      - skywalking
+
+exporters:
+  skywalking:
+    template:
+      source:
+        service: "{{ .Service.Name }}"
+        serviceInstance: "{{ .Pod.Name }}"
+        endpoint: ""
+      message: "{{ .Event.Message }}"
+    address: "127.0.0.1:11800" 
+```
+
+### Filter Config
+
+| name      | description                                                                                                                         | example          |
+|-----------|-------------------------------------------------------------------------------------------------------------------------------------|------------------|
+| reason    | Filter events of the specified reason, regular expression like `"Killing\|Killed"` is supported.                                    | `""`             |.                                                                                                           | `"127.0.0.1:11800"`                       |
```

Comment:
What's the meaning of `"127.0.0.1:11800"`?

Reply:
The TCP port of exposed by the oap service. I used the [default value](https://github.com/apache/skywalking-kubernetes-event-exporter/blob/9ad39c47eb347f13c0003d1ecd05c843c6d511ea/assets/default-config.yaml#L42), maybe I need to change it to a more reasonable address like `skywalking-system-oap.skywalking-system:11800`

Reply:
I didn't include a link to the [default config file](https://github.com/apache/skywalking-kubernetes-event-exporter/blob/main/assets/default-config.yaml#L42) because if you ignore the `EventExporter.Spec.Config` field, an empty string is used as configuration instead of the default config mentioned in skywalking-kubernetes-event-exporter. So there is no "default config" in EventExporter, the default is `""`. That's also why I used "example" for column name, not "default value".